tensorflow/python/ops/image_ops.py

```diff
@@ -1388,6 +1391,66 @@ def _crop_and_resize_shape(op):
 ops.RegisterShape('NonMaxSuppression')(common_shapes.call_cpp_shape_fn)
 
 
+def decode_image(contents, channels=None, ratio=None, fancy_upscaling=None,
+                 try_recover_truncated=None, acceptable_fraction=None,
```

Comment:
I think we should just drop the `ratio`, `fancy_upscaling`, `try_recover_truncated`, and `acceptable_fraction` arguments.  If you want these details you should use the underlying format op.